kernel/src/model/common.rs

```diff
@@ -41,10 +53,16 @@ static ALL_LANGUAGES: &[Language] = &[
 impl fmt::Display for Language {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let s = match self {
+            Self::Csharp => "C#",
+            Self::Dockerfile => "Dockerfile",
+            Self::Go => "Go",
+            Self::Java => "java",
             Self::JavaScript => "javascript",
+            Self::Json => "json",
             Self::Python => "python",
             Self::Rust => "rust",
             Self::TypeScript => "typescript",
+            Self::Yaml => "yaml",
```

Comment:
Some have capitals and other don't. Was there a reason for that?

Reply:
fixed!